packages/components/scripts/cleanReadmes.ts

```diff
@@ -86,7 +86,7 @@ const replaceEventsTable =
 
 // TODO: Add slots API
 const replaceSlotsTable =
-  (componentName: string): ((str: string) => string) =>
+  (): ((str: string) => string) =>
```

Comment:
double check

Reply:
Slots API not shown in storefront documentation yet, see TODO